tests/test_app/TestApp/Controller/CakesController.php

```diff
@@ -39,7 +39,7 @@ public function noRender()
     /**
      * invalid method
      *
-     * @return \Cake\Http\Response
+     * @return string
```

Comment:
The point of this kind of is to show that this is an invalid and undocumented return type.
Not sure if we should switch the documentation here.